docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<ul><li>... relative weights seem<s>s</s> preety good.</li><li>We can compute ESS values from the trace returned by <code>sample_smc</code>. <strong>Note that</strong> this is probably an overly optimistic value, as the computation of this ESS value<strong>s</strong> takes autocorrelation into account<strong>,</strong> <s>and</s> <strong>but</strong> each SMC run/chain has low autocorrelation by construction.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>This wasn't addressed in the latest commit</p>